### ISSUE TITLE
Request for Comment and Vote: Remove the proposal for an enforceable Code of Conduct

### ISSUE THREAD

**Issue by ianBanksia** [stacksgov/pm#199]
**Problem**
Issues #132 and #119 were created on the assumption we need a Code of Conduct. 
Part of the community believe this is against the "can't be evil" ethos.

**Stakeholders**
Stakeholders include all members of the Stacks community. Community members can be generally identified as members, contributors and leaders.

**Solution**
Scrap issue #132 altogether, either by closing it as SPAM or by starting a vote around the need for an enforceable Code of Conduct. 

Before assuming Stacks need it, it's important to take a step back and consult the community for review and decision making via voting mechanism as proposed by the Governance group grant proposal. https://github.com/stacksgov/Stacks-Grant-Launchpad/issues/27

**Comment by RagnarLifthrasir** [comment]
I strongly agree with removing the proposal for an enforceable Code of Conduct. The consensus is to remove it.
